includes/class-newspack-popups-model.php

```diff
@@ -1222,7 +1250,26 @@ class="<?php echo esc_attr( implode( ' ', $classes ) ); ?>"
 			</div>
 			<?php if ( ! $no_overlay_background ) : ?>
 				<div style="opacity: <?php echo floatval( $overlay_opacity ); ?>;background-color:<?php echo esc_attr( $overlay_color ); ?>;" class="newspack-lightbox-shim"></div>
-				</form>
+			<?php endif; ?>
+			<?php if ( 'form_submission' === $dismiss_action ) : ?>
+				<script data-amp-plus-allowed type="text/javascript">
```

Comment:
There's the `src/view` directory for front-end JS. Why should this feature rely on an inline script?